### PR TITLE
An update on the new course wizard

### PR DESCRIPTION
**Pull request by jesseyuen**
This is a short entry to document the status of the new courses wizard, who currently has access and what the plans for full release are.

### CODE REVIEW

**Review by fofr — CHANGES_REQUESTED** [review]
This page needs to document the reasons we didn't roll out New course to everyone. Please also tell the story of this feature, as it's been a long time coming.

Please add screenshots of the new course wizard as it currently looks on live.

**Comment by jesseyuen** [comment]
Good shout! I actually dont know much on the story of the feature beyond what is in the design history. Do you know if there was much scope change/updates during development? Please let me know :)

**Comment by jesseyuen** [comment]
I've fleshed this one out, added some links to associated research, previous wizard entries and also added some dates for the beginning and ending of the build phase to help try and tell this chapter of the story.

As a bonus while writing this I found a bug in the live wizard and also tidied up some of our internal processes that Support use for providers still using the google form :)

File: app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: The new course wizard is available for accredited bodies
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md]:
```suggestion
title: New course wizard as launched for accredited bodies
```

File: app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+## Access for accredited bodies
+
+Accredited bodies are able to access the new course wizard from the [Courses page](https://www.qa.publish-teacher-training-courses.service.gov.uk/organisations/T92/2020/courses).
+
+## Access for school directs
+
+Access to the wizard is yet been provided to school directs.
+
+## Why the wizard is not yet available to all users
+
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md]:
```suggestion
## Why the wizard is not yet available to school directs

```

File: app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+School directs will not receive access to the new course wizard until Publish can automatically notify them that a course they accredited has been created.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md]:
```suggestion
School directs will not receive access to the new course wizard until Publish can automatically notify their accredited body that a course has been created. 
```

File: app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+## Access for school directs
+
+Access to the wizard is yet been provided to school directs.
+
+## Why the wizard is not yet available to all users
+
+School directs will not receive access to the new course wizard until Publish can automatically notify them that a course they accredited has been created. 
+
+An accredited body needs to be notified immediately when courses they accredit are created so they can maintain accurate data in their record systems. This was highlighted in [Accredited bodies research - Reporting round 2](publish-teacher-training-courses/accredited-bodies-research-round-2).
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md]:
```suggestion
An accredited body needs to be notified immediately so they can maintain accurate data in their record systems. This was highlighted in [Accredited bodies research - Reporting round 2](publish-teacher-training-courses/accredited-bodies-research-round-2).
```

File: app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+ 
+Publish will shortly [provide these notifications](https://bat-design-history.netlify.com/publish-teacher-training-courses/notifications-mvp) to ensure this need is met. Once notifications are enabled all users will have access to the wizard.
+
+## Course creation for school directs
+
+All school direct users are required to [submit a google form](/publish-teacher-training-courses/new-course-google-form) to request a new course. 
+
+After recieving the request, DfE support creates the course manually and notifies both the school direct and associated accredited body via email of the new course’s availability.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-02-15-new-course-wizard-available.md]:
```suggestion
After receiving the request, DfE support creates the course manually and notifies both the school direct and associated accredited body via email of the new course’s availability.
```

**Review by fofr — CHANGES_REQUESTED** [review]
Nice. Just a few suggested tweaks to the content.

**Pull request merged**